app/src/main/java/com/cornellappdev/android/eatery/components/WaitTimesComponent.java

```diff
@@ -67,6 +75,43 @@ public void onClick(View v) {
         });
         mWaitTimesChart = view.findViewById(R.id.wait_time_chart);
         mWaitTimesChart.setNoDataText("");
+
+        mWaitTimesChart.setOnTouchListener(new View.OnTouchListener() {
+            @Override
+            public boolean onTouch(View v, MotionEvent event) {
+                if (mFingerJustTapped) {
+                    // Measure the yDiff directly after the tap, high y diff means motion was a
+                    // vertical swipe
+                    float yDiff = Math.abs(event.getY() - mLastY);
+                    float xDiff = Math.abs(event.getX() - mLastX);
+                    // If it was not a vertical swipe
```

Comment:
Do you think it'd be cleaner to do:
```
mVerticalScrolling = Math.abs(event.getY() - mLastY) - Math.abs(event.getX() - mLastX) > 0;
scrollView.requestDisallowInterceptTouchEvent(!mVerticalScrolling);
```

Reply:
Mmmm not exactly sure what "scrollView.requestDisallowInterceptTouchEvent(false)" would do and whether it would have any other side effects. Also I like the variables xDiff and yDiff just because while it may not be cleaner, I think it is more readable in terms of figuring out what is going on.

Reply:
Can def change tho if u guys think its better